admin-dev/themes/new-theme/js/pages/order/create/product-renderer.js

```diff
@@ -47,10 +47,16 @@ export default class ProductRenderer {
     }
 
     const $productsTableRowTemplate = $($(createOrderMap.productsTableRowTemplate).html());
+    const $productsTableGiftRowTemplate = $($(createOrderMap.productsTableGiftRowTemplate).html());
 
     for (const key in products) {
       const product = products[key];
-      const $template = $productsTableRowTemplate.clone();
+
+      let $template = $productsTableRowTemplate.clone();
```

Comment:
it's useless to clone twice, consume resource for nothing
```suggestion
      const $template = product.gift === true ? $productsTableGiftRowTemplate.clone() : $productsTableRowTemplate.clone();
```